### PR TITLE
chore(rust): grammar fix in ockam space help command

### LINKED ISSUE #6121: Fix grammar in `ockam space --help`
# Current behavior

Currently 2nd paragraph of text output from running `ockam space --help` produces this:
```
When you enroll in Ockam for the first time, a space is created for the user to host their projects.
```

Image

# Desired behavior 

Instead it should produce this:

```
When you enroll in Ockam for the first time, a space is created for you to host your projects.
```

Please take a look at this [file](https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/space/static/long_about.txt#L4) to get started.

---

<i>We love helping new contributors! ❤️
If you have questions or need help as you explore, please join us on [Discord](https://discord.gg/bewvnm6zqS). If you're looking for other issues to contribute to, please checkout our [good first issues](https://github.com/build-trust/ockam/labels/good%20first%20issue).</i>

### PR DESCRIPTION
**Pull request by shanan-P**
Resolves #6121

### CODE REVIEW

**Comment by mrinalwadhwa** [comment]
@pipliya Thank you for spending time on this, I noticed that you added your self to the CONTRIBUTORS file. To merge this pull request we need that to be separate pull request.

Would you mind creating a new pull request for that addition, please?

The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam/edit/develop/.github/CONTRIBUTORS.csv) file in the Github Web UI and create a pull request, this will mark the commit as verified.

File: implementations/rust/ockam/ockam_command/src/space/static/long_about.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 In Ockam, a space is an infinitely scalable Ockam Node in the cloud that can be created, managed, and scaled by Ockam Orchestrator. Spaces offer services such as creating projects within a space, inviting teammates to it, or attaching payment subscriptions.
 
-When you enroll in Ockam for the first time, a space is created for the user to host their projects.
+When you enroll in Ockam for the first time, a space is created for you to host your projects.
```
Comment (mrinalwadhwa) [inline implementations/rust/ockam/ockam_command/src/space/static/long_about.txt:3]: This change looks good, we can merge it right after your pull request accepting the CLA.

Comment (shanan-P) [reply implementations/rust/ockam/ockam_command/src/space/static/long_about.txt:3]: @mrinalwadhwa
I have created a separate pull request #6165 regarding this. 
Thank you!

**Review by mrinalwadhwa — APPROVED** [review]

**Comment by mrinalwadhwa** [comment]
@pipliya thank you and congratulations on an awesome first contribution 🎉🥳

**Comment by shanan-P** [comment]
Thank You @mrinalwadhwa 💯
I will be definitely looking here for more contributions🤠

**Pull request merged**